pkg/chains/tron/txm.go

```diff
@@ -0,0 +1,36 @@
+package tron
+
+import (
+	"fmt"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-evm/pkg/config"
+	"github.com/smartcontractkit/chainlink-evm/pkg/config/toml"
+	"github.com/smartcontractkit/chainlink-evm/pkg/keys"
+	tronkeystore "github.com/smartcontractkit/chainlink-tron/relayer/keystore"
+	tronclient "github.com/smartcontractkit/chainlink-tron/relayer/sdk"
+	trontxm "github.com/smartcontractkit/chainlink-tron/relayer/txm"
+)
+
+func ConstructTronTxm(logger logger.Logger, cfg *config.ChainScoped, nodes []*toml.Node, keystore keys.Store) (*trontxm.TronTxm, error) {
```

Comment:
Can probably drop `Tron` from the name given we are in `package tron`:
```suggestion
func ConstructTxm(logger logger.Logger, cfg *config.ChainScoped, nodes []*toml.Node, keystore keys.Store) (*trontxm.TronTxm, error) {
```
Who was the original author? Can we include a comment here about what is going on exactly?

Reply:
@KodeyThomas could you please suggest a comment explaining this code?

Reply:
```suggestion
// Creates the Transaction Manager used for Tron-based chains. Uses an adapter of the EVM Keystore to sign.
// Doesn't currently support multinode writes
```

Reply:
Thanks, I have updated the method name and the comment.